framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -864,17 +990,36 @@ private AnnotationMirror createResultingAnnotation(TypeMirror resultType, List<?
      * @param values list of longs; duplicates are allowed and the values may be in any order
      * @return a {@link IntVal} annotation using the values
      */
-    public AnnotationMirror createIntValAnnotation(List<Long> values) {
-        if (values == null) {
+    public AnnotationMirror createIntValAnnotation(List<Long> intValues) {
```

Comment:
The Javadoc for the method should be updated, as it can now also return an IntRange. Maybe a different name for the method would also be good?